docs/topics/dynamic-content.rst

```diff
@@ -248,8 +248,38 @@ Using a headless browser
 A `headless browser`_ is a special web browser that provides an API for
 automation.
 
-The easiest way to use a headless browser with Scrapy is to use Selenium_,
-along with `scrapy-selenium`_ for seamless integration.
+Since version 2.0, it is possible to integrate libraries that use the
+``async/await`` syntax. One such library is `pyppeteer`_ (an unnoficial
+Python port of `puppeteer`_), which uses headless Chrome to download and
+render pages.
```

Comment:
I would not mention coroutine syntax or the Scrapy version that introduced it. I would simply state here that installing the asyncio reactor (with a link to the corresponding section, which explains that support was added in Scrapy 2.0) allows you to use pyppeteer, an asyncio-based headless Chrome/Chromium automation library.

Doing this also makes the first post-example paragraph unnecessary.